Examples/Tests/Langmuir/PICMI_inputs_langmuir_rz_multimode_analyze.py

```diff
@@ -84,8 +84,10 @@
                              n_azimuthal_modes = 3,
                              lower_bound = [rmin, zmin],
                              upper_bound = [rmax, zmax],
-                             lower_boundary_conditions = ['dirichlet', 'periodic'],
-                             upper_boundary_conditions = ['dirichlet', 'periodic'],
+                             lower_boundary_conditions = ['pec', 'periodic'],
```

Comment:
Currently, 'pec' is not a standard picmi specification. This should remain 'dirichlet' and it should be mapped to 'pec' in picmi.py.

Reply:
cc @RevathiJambunathan we should probably standardize this timely in PICMI as well, now that we have found a good convention for your PR :)